### DISCUSSION TITLE
Using .onnx files with Yolov7 in Norfair

### DISCUSSION THREAD

**Discussion by moooises** [tryolabs/norfair#236] [Q&A] [answered]
Hello,

Anyone knows if there is a way to use .onnx files in YoloV7?
I have seen some [examples ](https://github.com/ibaiGorordo/ONNX-YOLOv7-Object-Detection/blob/248dab0e87f0f1287da39ee9b1f3263f22205d6f/yolov7/YOLOv7.py#L9) using **onnxruntime**, but the class YoloV7 made by them is quite different from the one in the demos. So I'm not sure if this option will work with Norfair.

**Comment by facundo-lezama** [comment, marked as answer]
Hi @moooises! Norfair is model agnostic, meaning you may use whatever detector you want. The only thing Norfair needs you to do is to use `Detection` objects. So you may change the YOLOv7 class as you wish and then be careful to translate the model's output to `Detection` objects.
